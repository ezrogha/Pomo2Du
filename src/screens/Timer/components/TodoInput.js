import React from 'react'
import { StyleSheet, View, TextInput, Dimensions, Platform } from 'react-native'

const { height, width } = Dimensions.get('window');

export default ({ text, onChange }) => (
    <View style={styles.textContainter}>
        <TextInput style={styles.textInput} placeholder="Enter Task name..." value={text} onChangeText={onChange} />
    </View>
)

const styles = StyleSheet.create({
    textContainter: {
        top: Platform.OS === "ios" ? height/6 : 60,
        position: "absolute",
        zIndex: 50
    },
    textInput: {
        fontSize: 18,
        color: "grey"
    }
})