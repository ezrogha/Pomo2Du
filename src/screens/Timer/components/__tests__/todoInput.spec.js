import React from 'react'

import TodoInput from '../TodoInput';
import renderer from 'react-test-renderer'

describe("Testing TodoInput Component", () => {
    test("renders correctly", () => {
        const tree = renderer.create(<TodoInput />).toJSON()
        expect(tree).toMatchSnapshot()
    })
})
