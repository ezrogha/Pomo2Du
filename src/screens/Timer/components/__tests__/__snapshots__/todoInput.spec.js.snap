// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing TodoInput Component renders correctly 1`] = `
<View
  style={
    Object {
      "position": "absolute",
      "top": 222.33333333333334,
      "zIndex": 50,
    }
  }
>
  <TextInput
    allowFontScaling={true}
    placeholder="Enter Task name..."
    rejectResponderTermination={true}
    style={
      Object {
        "color": "grey",
        "fontSize": 18,
      }
    }
    underlineColorAndroid="transparent"
  />
</View>
`;
